README.md

```diff
@@ -500,12 +500,20 @@ gem "image_optim_bin" # Optional
 
 #### Configuration
 
+There is a default preset named 'jekyll' defined in `lib/jekyll/assets/plugins/proxy/optim.rb`, you can extend it or add new ones in assets.plugins.img.optim configuration branch:
+
 ```yml
 assets:
   plugins:
     img:
       optim:
-        {}
+        jekyl:
```

Comment:
Jekyll is spelled wrong.  Aside from that it's unacceptable to override my defaults in the documentation about my defaults.  If you do not like the defaults please file a pull request and explain why it's necessary to change them, rather then overriding them in the examples, creating a contradictory sentiment.